Earthfile

```diff
@@ -20,7 +21,7 @@ END
 ARG COSIGN_EXPERIMENTAL=0
 ARG CGO_ENABLED=0
 # renovate: datasource=docker depName=quay.io/kairos/osbuilder-tools versioning=semver-coerced
-ARG OSBUILDER_VERSION=v0.5.3
+ARG OSBUILDER_VERSION=v0.6.0
```

Comment:
this version has support for multiarch packages for elemental and livecd grub artifacts. So using an arm64 image will have the proper artifacts for that arch on it.